### PR TITLE
chore(scss): added stylelint

### PR DESCRIPTION
**Pull request by DalderupMaurice**
## Description
Stylelint to follow scss conventions, resulting in clean code
It is also added to the CI

## Motivation and Context
read above

## How Has This Been Tested?
yarn stylelint

## Types of changes
- [x] Chore (tests, refactors, and fixes)
- [ ] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
- [ ] Tests for the changes have been added (for bug fixes/features)

#### Documentation
- [ ] Docs need to be added/updated (for bug fixes/features)

## Closing issues
Fixes #

### CODE REVIEW

**Comment by DalderupMaurice** [comment]
@mhuggins 
I have the feeling there are not enough errors 😂 It's also not "seeing" the quotes missing at the background-image tags. I am not sure if this is a rule or included in any config :P 

Currently the flex issue is not resolved aswell (Tests will fail xD)

File: src/renderer/root/components/AuthenticatedLayout/AuthenticatedLayout.scss [hunk 1]
```diff
@@ -74,7 +72,8 @@ $sidebar-shadow: inset -1px 0 0 #d7d8de;
         flex: 1 1 auto;
         box-sizing: border-box;
         overflow-y: auto;
-        flex: 1;
+        height: 100vh;
+        //flex: 1;
```
Comment (DalderupMaurice) [inline src/renderer/root/components/AuthenticatedLayout/AuthenticatedLayout.scss]: You should verify if this is okay :P (Duplicate `flex 1` which resulted in the page height being fullscreen with `height: 100vh`

Comment (mhuggins) [reply src/renderer/root/components/AuthenticatedLayout/AuthenticatedLayout.scss]: Try `flex: 1 1 100%;`

Comment (mhuggins) [reply src/renderer/root/components/AuthenticatedLayout/AuthenticatedLayout.scss]: This should be sufficient:

```js
.content {
  flex: 1 1 100%;
  overflow-y: auto;
  box-sizing: border-box;
}
```

Comment (DalderupMaurice) [reply src/renderer/root/components/AuthenticatedLayout/AuthenticatedLayout.scss]: done :P Fixed it

**Comment by DalderupMaurice** [comment]
Also rearranged the order of deps in this PR @mhuggins

**Pull request merged**